src/main/java/frc/robot/subsystems/elevatorIO/ElevatorIOSparks.java

```diff
@@ -87,11 +93,36 @@ public void updateInputs(ElevatorInputs inputs) {
     inputs.velocityInchesPerSecondRight = right.getEncoder().getVelocity();
     inputs.tempCelsiusRight = right.getMotorTemperature();
     inputs.currentDrawAmpsRight = right.getOutputCurrent();
+
+    currentLeftFF = leftFF.calculate(inputs.velocityInchesPerSecondLeft);
+    currentRightFF = rightFF.calculate(inputs.velocityInchesPerSecondRight);
   }
 
-  @Override
-  public void setVoltage(double volts) {
-    left.setVoltage(volts);
-    right.setVoltage(volts);
+  public void setPIDFF() {
```

Comment:
Does this actually get called anywhere?

Reply:
Fixed in https://github.com/FRC2713/Robot2023/pull/77/commits/902c5a2d295ef635b6cba7075d53b0b53b197969